ui/index.jsx

```diff
@@ -94,6 +87,11 @@ if (process.env.SEARCH_API_URL) {
   setSearchApi(process.env.SEARCH_API_URL);
 }
 
+// Fix to make sure old users' cookies are set to the correct domain
+// This can be removed after March 11th, 2021
```

Comment:
Not sure when it's ok to remove this

Reply:
since this isn't a general migration framework, you could call this something like `doCookieCleanupMarch2020()`